deltachat-ffi/deltachat.h

```diff
@@ -4099,6 +4099,16 @@ char*           dc_contact_get_profile_image (const dc_contact_t* contact);
 uint32_t        dc_contact_get_color         (const dc_contact_t* contact);
 
 
+/**
+ * Get the contact's status.
+ *
+ * Status is the last signature received in a message from this contact.
+ * @return Contact status, if any.
```

Comment:
```suggestion

 * @memberof dc_contact_t
 * @param contact The contact object.
 * @return Contact status, if any.
```

this boilerplate is needed for doxygen.